package.json

```diff
@@ -52,6 +52,7 @@
         "express": "^4.16.2",
         "helmet": "^3.9.0",
         "jsonwebtoken": "^8.1.1",
+        "leaflet-easybutton": "^2.3.0",
```

Comment:
Maybe I'm missing something but I think this needs to be installed as a dependency for the client side not the server side

Reply:
Ah.. I completely missed it, Yes you are right it should be in Client. Thanks @jkwening . I think I was too lost in directions to notice it :-)